### PR TITLE
add apbs package

### PR DESCRIPTION
**Pull request by bgruening**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bgruening** [comment]
ping @simonbray

File: recipes/apbs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        - ncurses
+        - bzip2
+
+test:
+    commands:
+        - apbs --version || test $? -eq 13
+
+about:
+    home: http://www.poissonboltzmann.org/
+    license: BSD-like
```
Comment (chrisburr) [inline recipes/apbs/meta.yaml]:
```suggestion
    license: BSD-3-Clause
```

File: recipes/apbs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    git_url: https://github.com/Electrostatics/apbs-pdb2pqr.git
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
```
Comment (chrisburr) [inline recipes/apbs/meta.yaml]:
```suggestion
        - cmake
        - make
```

File: recipes/apbs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{% set name = "apbs" %}
+{% set version = "1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    git_rev: apbs-{{ version }}
```
Comment (chrisburr) [inline recipes/apbs/meta.yaml]: Can you build from a tarball instead? See: http://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos

File: recipes/apbs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    git_url: https://github.com/Electrostatics/apbs-pdb2pqr.git
+
+build:
+    number: 0
```
Comment (chrisburr) [inline recipes/apbs/meta.yaml]: It looks like you don't include a `build.bat` file so Windows should be skipped
```suggestion
    number: 0
    skip: true  # [win]
```

File: recipes/apbs/meta.yaml (hunk 3 above)
Comment (bgruening) [reply recipes/apbs/meta.yaml]: They are using sub.modules intensively and the provided release tarball does not contain those sources.

Comment (chrisburr) [reply recipes/apbs/meta.yaml]: That makes, maybe make an issue about it upstream if there isn't already one?

**Comment by bgruening** [comment]
Thanks @chrisburr

**Pull request merged**